contracts/JBPrices.sol

```diff
@@ -53,7 +58,9 @@ contract JBPrices is IJBPrices, Ownable {
     AggregatorV3Interface _feed = feedFor[_currency][_base];
 
     // Feed must exist.
-    require(_feed != AggregatorV3Interface(address(0)), '0x03: NOT_FOUND');
+    if (_feed == AggregatorV3Interface(address(0))) {
+      revert JBErrors.NOT_FOUND();
```

Comment:
@drgorillamd @dhyon @mejango Should we remove these generic errors and make everything context-specific?

I.e., PRICE_FEED_NOT_FOUND? The mix of JBErrors and localized errors is throwing me off – even though I know I am reneging on my original stance.